CHANGELOG.md

```diff
@@ -4,8 +4,17 @@ parent: README
 nav_order: 1
 ---# Changelog
 
+## [2.1.5] - 2021-01-21
+- Add support for renaming projects
+- Use latest go-client
```

Comment:
Since this doesn't in itself mean anything I'd remove this or if the new client version actually fixed some issues then explicitly mention what was fixed.